src/MetaModels/Attribute/Longtext/Longtext.php

```diff
@@ -38,6 +41,22 @@ public function getSQLDataType()
     }
 
     /**
+     * @{inheritDoc}
+     *
+     * add a fulltext-index to column after creation
+     */
+    public function createColumn()
+    {
+        parent::createColumn();
+
+        TableManipulation::addIndex(
```

Comment:
Do we have to drop this index prior deleting the column?
If so, we will need to override the deleteColumn() as well.